src/components/Select/Select.stories.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Select from './Select';
+
+export default {
+  title: 'Components/Select',
+  component: Select
+};
+
+export const HelloWorld = () => <Select value="Example label" />;
```

Comment:
Update this file to show a few variations using the different props you add in the main jsx file.